controller/api/apis.go

```diff
@@ -1769,6 +1771,8 @@ type RESTSystemConfigConfig struct {
 	ScannerAutoscale          *RESTSystemConfigAutoscaleConfig `json:"scanner_autoscale,omitempty"`
 	NoTelemetryReport         *bool                            `json:"no_telemetry_report,omitempty"`
 	RemoteRepositories        *[]RESTRemoteRepository          `json:"remote_repositories,omitempty"`
+	EnableTLSVerification     *bool                            `json:"enable_tls_verification"`
```

Comment:
According to golang doc, this option only affects encoding.

> The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.

Since the struct is only used to parse REST API input, i.e., decoding from JSON, it should be alright, but I will change it to keep consistency.  Thanks!